packages/cozy-pouch-link/src/PouchManager.js

```diff
@@ -298,6 +298,7 @@ class PouchManager {
       localStorage.persistWarmedUpQueries(this.warmedUpQueries)
       logger.log('PouchManager: warmupQueries for ' + doctype + ' are done')
     } catch (err) {
+      logger.error('Could not warm up queries', err)
```

Comment:
nit: Peut-être mettre 'PouchManager: Could not warm up queries' pour rester cohérent avec ce qu'on a au-dessus ?

Reply:
Ah oui bien vu, merci, c'est fait.